code/game/golf/golf.dm

```diff
@@ -78,11 +78,13 @@
 	icon_state ="golfball"
 	throwforce = 12
 	attack_verb = list("hit")
+	var/last_hit = 0
```

Comment:
Whats up with this. doesnt seem to be actually used

Reply:
forgot to remove it
```suggestion

```